rubocop-govuk.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "rake", "~> 12.0"
 
-  spec.add_dependency "rubocop", "~> 0.76"
+  spec.add_dependency "rubocop", "0.77"
```

Comment:
Consistency between apps and version is good. But do we have an agreement for when in the future we bump this? What's stopping us getting really far behind?

Reply:
Ah so my assumption is we bump it along with Rubocop. Was looking at adding Dependabot to help keep it up to date.